cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -3653,9 +3653,9 @@ pub(crate) fn define(
             .build(),
     );
 
-    let lo = &operand("lo", NarrowInt);
-    let hi = &operand("hi", NarrowInt);
-    let a = &operand_doc(
+    let lo = &Operand::new("lo", NarrowInt);
+    let hi = &Operand::new("hi", NarrowInt);
+    let a = &Operand::with_doc(
```

Comment:
Just looking at this makes me think: are we going to do the same thing for instructions that we did with formats and immediates?

Reply:
Probably, yeah. Instructions need to know one final index (OpcodeNumber), so it's not trivial as it was for immediates, and might require some working around like we did for formats.